src/orquestra/sdk/shared/_spaces/_structs.py

```diff
@@ -3,7 +3,7 @@
 ################################################################################
 from dataclasses import dataclass
 
-from orquestra.sdk.schema.workflow_run import ProjectId, WorkspaceId
+from orquestra.sdk.shared.schema.workflow_run import ProjectId, WorkspaceId
```

Comment:
**thought**: if we agree to only use relative imports within the same to-be-wheel, we'll have to change this and similar imports, too.